pkg/registry/common/expire/nse_server.go

```diff
@@ -32,35 +32,38 @@ import (
 )
 
 type expireNSEServer struct {
-	nseExpiration time.Duration
-	ctx           context.Context
+	defaultNseExpiration time.Duration
```

Comment:
Should we get rid of default expiration? Seems like the update path is going to always put the expiration time for nses after these changes.

```suggestion
```

For unit tests, we can add `injectexpirationtime` chain element.

Reply:
Totally agree
But should we consider the situation where the user hasn't `updatepath`?